app/controllers/procurement_areas_controller.rb

```diff
@@ -0,0 +1,43 @@
+class ProcurementAreasController < ApplicationController
+  def index
+    @procurement_areas = ProcurementArea.ordered_by_name
+  end
+
+  def new
+    @procurement_area = ProcurementArea.new
+  end
+
+  def create
+    procurement_area = ProcurementArea.new(procurement_area_params)
+
+    if procurement_area.save
+      redirect_to procurement_areas_path
+    end
+  end
```

Comment:
Yes, my bad. I wanted to get feedback on the PR, so I rushed a bit. Will do that, thanks :)